remote-config.jsonc

```diff
@@ -99,7 +96,7 @@
           "message": "Sign up for the DDEV newsletter! https://ddev.com/newsletter/"
         },
         {
-          "message": "If you miss phpMyAdmin in DDEV, run `ddev get ddev/ddev-phpmyadmin`, or try one of the many other database browsers, https://ddev.readthedocs.io/en/stable/users/usage/database-management/#database-guis"
+          "message": "For phpMyAdmin in DDEV, run `ddev get ddev/ddev-phpmyadmin`, or try one of the many other database browsers, https://ddev.readthedocs.io/en/stable/users/usage/database-management/#database-guis"
```

Comment:
"To add phpMyAdmin in DDEV,..."? and i wonder if it would make sense to strike the "or try" part? the tip is about phpmyadmin why bloat it up with information about additional databases? maybe move the part about other databases to a separate tip?